iwf/command_request.py

```diff
@@ -19,12 +19,10 @@ class TimerCommand:
     duration_seconds: int
 
     @classmethod
-    def timer_command_by_duration(
-        cls, duration: timedelta, command_id: Optional[str] = None
-    ):
+    def by_seconds(cls, duration_seconds: int, command_id: Optional[str] = None):
```

Comment:
rename this method as I realized that's a bit awkward to use `TimerCommand.timer_command_by_duration(...)`, and renamed to `TimerCommand.by_seconds()` looks more concise. 

I will put this as a breaking changes in the release.
Luckily our internal users didn't use this API. they use the constructor directly